README.md

```diff
@@ -18,7 +18,7 @@ Once the SDK is installed, the operator can be built via:
 ```console
 $ dep ensure --vendor-only
 
-$ operator-sdk build quay.io/openshift/ocs-operator
+$ make operator-sdk
```

Comment:
should this be `make ocs-operator` ?

Reply:
You're correct. I'll fix it.

This was just me trying to optimize how much I type. And now, I've typed so much more.